dht.go

```diff
@@ -284,44 +280,29 @@ func (dht *IpfsDHT) FindLocal(id peer.ID) pstore.PeerInfo {
 }
 
 // findPeerSingle asks peer 'p' if they know where the peer with id 'id' is
-func (dht *IpfsDHT) findPeerSingle(ctx context.Context, p peer.ID, id peer.ID) (*pb.Message, error) {
-	eip := log.EventBegin(ctx, "findPeerSingle",
-		logging.LoggableMap{
-			"peer":   p,
-			"target": id,
-		})
-	defer eip.Done()
+func (dht *IpfsDHT) findPeerSingle(ctx context.Context, p peer.ID, id peer.ID) (_ *pb.Message, _err error) {
+	ctx = dht.logSpan(ctx, "findPeerSingle")
+	defer func() {
+		log.FinishWithErr(ctx, _err)
+	}()
+
+	log.SetTag(ctx, "peer", p)
+	log.SetTag(ctx, "target", p)
```

Comment:
here again. I very much liked the old `defer EventBegin().Done()` pattern, can't we replicate with the new logger?
It's a loss of ergonomics to not have it.